test-framework/junit5-internal/src/main/java/io/quarkus/test/QuarkusProdModeTest.java

```diff
@@ -458,16 +468,22 @@ private void startBuiltResult(Path builtResultArtifact) throws IOException {
         List<String> command = new ArrayList<>(systemProperties.size() + 3);
         if (builtResultArtifact.getFileName().toString().endsWith(".jar")) {
             command.add(JavaBinFinder.findBin());
+            if (this.jvmArgs != null) {
+                command.addAll(this.jvmArgs);
+            }
             command.addAll(systemProperties);
             command.add("-jar");
             command.add(builtResultArtifact.toAbsolutePath().toString());
         } else {
             command.add(builtResultArtifact.toAbsolutePath().toString());
+            if (this.jvmArgs != null) {
+                command.addAll(this.jvmArgs);
+            }
             command.addAll(systemProperties);
         }
 
         command.addAll(Arrays.asList(commandLineParameters));
-
+        System.out.println("Launching " + command);
```

Comment:
Is it something we should remove or you really want to print the command?

Reply:
I'll remove it. It's a leftover.